sdk/core/azure-core-tracing-opentelemetry/src/test/java/com/azure/core/tracing/opentelemetry/OpenTelemetryTracerTest.java

```diff
@@ -49,12 +48,7 @@
 import static com.azure.core.util.tracing.Tracer.SPAN_CONTEXT_KEY;
 import static com.azure.core.util.tracing.Tracer.USER_SPAN_NAME_KEY;
 import static io.opentelemetry.api.trace.StatusCode.UNSET;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
`*` imports will be flagged by checkstyle.

Reply:
they aren't - all pipeline is green a this's IntelliJ conveniently changing them to `*`. Anyway, returned them back.

Reply:
Ah, apparently they are allowed in test files, but I haven't seen that usually.
https://github.com/Azure/azure-sdk-for-java/blob/main/eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml#L37